### PR TITLE
Documentation for Dynamic Configuration

### PR DESCRIPTION
**Pull request by underhood**
##### Summary
Documents Dynamic configuration

 - [x] readme.md

##### Additional Information

<details> <summary>For users: How does this change affect me?</summary>
  
</details>

### CODE REVIEW

**Comment by andrewm4894** [comment]
wondering if the docs might make more sense in here: https://github.com/netdata/netdata/tree/master/docs/configure

not sure what we usually do - think it depends a bit.

just that `libnetdata/dyn_conf` feels maybe a little off beaten track if i'm just scanning the github id prob never go here and so not discover concept of dynamic config 

not sure but just mentioning

File: libnetdata/dyn_conf/README.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+agent->plugin:
+=============
+FUNCTION_PAYLOAD [UUID] 1 "set_plugin_config"
+the new configuration
+blah blah blah
+FUNCTION_PAYLOAD_END
+
+plugin->agent:
+=============
+FUNCTION_RESULT_BEGIN [UUID] [(0/1)(accept/reject)] [text/plain] 5
```
Comment (ilyam8) [inline libnetdata/dyn_conf/README.md:59]:
```suggestion
FUNCTION_RESULT_BEGIN [UUID] [(1/0)(accept/reject)] [text/plain] 5
```

**Comment by Ancairon** [comment]
Where do we want this file in Learn? I think I would put it under `Configuring`.

**Comment by underhood** [comment]
i cannot approve as an PR author
thanks @ilyam8 for making it nicer

File: libnetdata/dyn_conf/README.md [hunk 2]
```diff
@@ -1,51 +1,74 @@
...
+
 ### Plugin level
-- `GET .../[plugin_name]` - gets the plugins current configuration from the plugin using `get_plugin_config` command
-- `PUT .../[plugin_name]` - updates the plugin configuration
-- `GET .../[plugin_name]/modules` - get list of modules registered by plugin and their types
-- `GET .../[plugin_name]/schema` - gets configuration schema of the plugin
+
+| Method  | Endpoint                          | Description                  |
+|:-------:|-----------------------------------|------------------------------|
+| **GET** | `/api/v2/config/[plugin]`         | Plugin config                |
```
Comment (ilyam8) [inline libnetdata/dyn_conf/README.md:29]: @underhood I find it much easier to follow without always having in mind that "this is not a full endpoint but relative to smth", so decided to add `/api/v2/config` to all endpoints.

**Review by ilyam8 — APPROVED** [review]
Some things are missing (e.g. DYNCFG_REGISTER_JOB, REPORT_JOB_STATUS). They will be added in a separate PR.

**Review by Ancairon — APPROVED** [review]

**Comment by underhood** [comment]
rebased on master and fixed the CLA no other changes

**Comment by underhood** [comment]
@Ancairon this is relevant to people who want to develop plugins not to regular users at this point

**Pull request merged**